common/views/themes/utility-classes.ts

```diff
@@ -42,6 +43,33 @@ export const utilityClasses = css<GlobalStyleProps>`
     `}
   }
 
+  // This removes the element from the flow, as well as it's visibility
```

Comment:
nit: I don't think we want an apostrophe here?

```suggestion
  // This removes the element from the flow, as well as its visibility
```